riskiq/api.py

```diff
@@ -1002,41 +1004,42 @@ def post_inventory_search(self, query=None, filter=None, filters=None):
 
     def get_v2_events(self, days=1, start=None, end=None, count=50, offset=0):
```

Comment:
What is the difference between a V1 event and a V2 event? There should only be Events (no versions), and hence there should be no _v2_ in the method name.